### PR TITLE
query and update animals table

### PR DESCRIPTION
**Pull request by baqar-abbas**
In the project Milestone -:

-     Addition of species column.
-     Update, delete and aggregate functions
-     Insertion of data to animals table
-     Action code copied to data.sql
-     SQL transactions to commit, rollback and save points
-     Screen shots displayed below:
select1
query1
query2
query3
query4
query5
query6
query7
query8
query9
query10
query11
query12
query13
query14
query15

### CODE REVIEW

File: data.sql [hunk 1]
```diff
@@ -5,3 +5,14 @@ VALUES ('Agumon', DATE '2020-02-03', 0, TRUE, 10.23),
        ('Gabumon', DATE '2018-11-15', 2, TRUE, 8),
        ('Pikachu', DATE '2021-01-07', 1, FALSE, 15.04),
        ('Devimon', DATE '2017-05-12', 5, TRUE, 11);
+
+       /* Project milestone 2 */
+
+       INSERT INTO animals(name, date_of_birth, escape_attempts, nuetered, weight_kg)
+       VALUES ('Charmander', DATE '2020-02-08', 0, FALSE, 11),
```
Comment (mukhammadrizooff) [inline data.sql:12]: - [ ] [REQUIRED] Good job so far! Please make sure to value for `weight_kg`, it should be `-11` not a `11` 🙏🏼 

```sql
    INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
    VALUES ('Charmander', '2020/2/8', '0', 'false', '-11.0');
```

**Review by mukhammadrizooff — CHANGES_REQUESTED** [review]
Hi @baqar-abbas, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights 
- Correct GitFlow ✔️ 
- Good implementations ✔️
- Descriptive `README` file ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: data.sql (hunk 1 above)
Comment (baqar-abbas) [reply data.sql:12]: Suggested changes have been made screen shot attached for your kind reference. 
reviewquery

Comment (baqar-abbas) [reply data.sql:12]: sending it again for a re review

**Review by prolajumokeoni — APPROVED** [review]
Hi @baqar-abbas 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
## Highlights
 - All changes requested have been implemented 🥇  
## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by baqar-abbas** [comment]
Thanks :)

**Pull request merged**